1: New Select Features (#216)
sofarsounds/maestro: src/atoms/MenuItem/README.md

# MenuItem

To implement a `MenuItem` wrapper into your project you'll need to add this import
```js
import { MenuItem } from '@sofarsounds/maestro'
```

> After adding the import you can use any button simply like this

Author: ```suggestion
After adding the import you can use a MenuItem simply like this
```

```html

> <MenuItem>I am beautiful</Menu>

Reviewer: <3
